README.md

```diff
@@ -0,0 +1,10 @@
+How to do a release
+===================
+
+- Change the version numbers in `versions.json` to the versions you want to
+  release. Keep the versions of modules you don't want to release unchanged.
+- Run `release.raku --prepare`. This will do a release in all modules that have
```

Comment:
I would replace "This will do a release..." with a description of the most important things that will happen:

* modifications to files
* git operations like clone, tag, etc.